Misc/NEWS.d/next/Library/2025-12-08-18-12-44.gh-issue-142438.UF_0nd.rst

```diff
@@ -0,0 +1,2 @@
+Added a missing GIL release in _PySSL_keylog_callback when keylog_bio is
+unset, preventing a leaked GIL in that early-return path.
```

Comment:
We could (1) either remove the NEWS entry or (2) indicate what could be wrong from a public API perspective. I don't want to expose implementation details like this.